examples/decoding/plot_haxby_space_net.py

```diff
@@ -0,0 +1,62 @@
+"""
+Decoding with SpaceNet: face vs house object recognition
+=========================================================
+
+Here is a simple example of decoding with a SpaceNet prior (i.e S-LASSO,
+TV-l1, etc.), reproducing the Haxby 2001 study on a face vs house
+discrimination task.
+"""
+
+### Load Haxby dataset ########################################################
+from nilearn.datasets import fetch_haxby
+data_files = fetch_haxby()
+
+### Load Target labels ########################################################
+import numpy as np
+labels = np.recfromcsv(data_files.session_target[0], delimiter=" ")
+
+
+### Split data into train and test samples ####################################
+target = labels['labels']
+condition_mask = np.logical_or(target == "face", target == "house")
+condition_mask_train = np.logical_and(condition_mask, labels['chunks'] <= 6)
+condition_mask_test = np.logical_and(condition_mask, labels['chunks'] > 6)
+
+### make X (design matrix) and y (response variable)
+import nibabel
+from nilearn.image import index_img
+niimgs  = nibabel.load(data_files.func[0])
+X_train = index_img(niimgs, condition_mask_train)
+X_test = index_img(niimgs, condition_mask_test)
+y_train = target[condition_mask_train]
+y_test = target[condition_mask_test]
+
+
+### Loop over Smooth-LASSO and TV-L1 penalties ###############################
```

Comment:
We should rename Smooth-Lasso everywhere to GraphNet. I am sorry, it's a lot of work, but we need to do that to make sure that we really earn the good will of the community.